shared/modules/types.go

```diff
@@ -48,6 +48,11 @@ type PersistenceConfig interface {
 	GetBlockStorePath() string
 	GetTxIndexerPath() string
 	GetTreesStoreDir() string
+	GetMaxConnsCount() int32
+	GetMinConnsCount() int32
+	GetMaxConnLifetimeHour() uint64
+	GetMaxConnIdleTimeMinute() uint64
+	GetHealthCheckPeriodMinute() uint64
```

Comment:
I looked into: https://pkg.go.dev/github.com/jackc/pgx/v4/pgxpool

And found that it uses found time.Duration: https://pkg.go.dev/time#Duration

It seems that we can easily parse things like `3s` or `1h` in golang: https://www.geeksforgeeks.org/time-parseduration-function-in-golang-with-examples/

Thoughts on removing the `Hour`, `Minute`, etc.., but making these strings so it could be configured to seconds, minutes, hours, etc in the config level.

This will also be similar to how tendermint designed things: github.com/pokt-network/pocket

![Screenshot 2022-12-21 at 12 24 35 PM](https://user-images.githubusercontent.com/1892194/208966653-1ace0963-67b5-42f2-ac60-46e333d32334.png)

Reply:
Makes sense, will make changes accordingly